deny.toml

```diff
@@ -52,6 +52,12 @@ skip = [
     {name = "windows_x86_64_msvc" },
     {name = "curve25519-dalek"},
     {name = "hashbrown"},
+    {name = "universal-hash"},
+    {name = "poly1305"},
+    {name = "cipher"},
+    {name = "chacha20poly1305"},
+    {name = "chacha20"},
+    {name = "aead"},
```

Comment:
It is possible some item can now be removed from this list and other cargo deny "exception" lists

Reply:
Same idea occurred to me... but honestly I don't care right now. Maybe eventually we can right a script to check that.